public/src/app/app.component.html

```diff
@@ -194,12 +194,7 @@
           <td class='issues'>
 
             <ng-container *ngFor="let milestone of site.value.milestones">
-              <ng-container *ngIf="site.key != 'developer.keyman.com'">
-                <app-count-box [repo]="site.key" [title]="milestone.title" [count]="milestone.count" [class]="'milestone-'+milestone.id"></app-count-box>                
-              </ng-container>
-              <ng-container *ngIf="site.key == 'developer.keyman.com'">
-                <app-count-box [repo]="site.key" [title]="milestone.title" [count]="milestone.count" [class]="'milestone-future'"></app-count-box>                
-              </ng-container>
+              <app-count-box [repo]="site.key" [title]="milestone.title" [count]="milestone.count" [class]="'milestone-'+milestone.id"></app-count-box>
```

Comment:
When developer.keyman.com now in sprint cycle, shouldn't we see @ermshiperete in the contributions bar for
keymanapp/developer.keyman.com#37 ?

Reply:
We will see him in the contributions bar when he opens a PR this cycle. His last PR was on Friday last week.